File: C#/TopflytechCodec/TopflytechCodec/Entities/ConfigMessage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace TopflytechCodec.Entities
{
    /// <summary>
    /// The type Config message.When the server sends a configuration command to the device,
    /// the device returns the result of the setup. This class describes the setting result
    /// </summary>
    public class ConfigMessage : Message  
    {
        private String configResultContent;
	    public String ConfigResultContent
	    {
		    get { return configResultContent; }
		    set { configResultContent = value; }
	    }

    }
}

File: Java/topflytechCodec/src/com/topflytech/codec/entities/ConfigMessage.java
package com.topflytech.codec.entities;
 

/**
 * The type Config message.When the server sends a configuration command to the device,
 * the device returns the result of the setup. This class describes the setting result
 */
public class ConfigMessage  extends Message{

    /**
     * Gets config result content.
     *
     * @return the config result content
     */
    public String getConfigResultContent() {
        return configResultContent;
    }

    /**
     * Sets config result content.
     *
     * @param configResultContent the config result content
     */
    public void setConfigResultContent(String configResultContent) {
        this.configResultContent = configResultContent;
    }

    /**
     * The Config result content.
     */
    protected String configResultContent;
}
